spec/support/webmock.rb

```diff
@@ -2,18 +2,25 @@
 [:delete, :get, :post, :put, :patch].each do |method|
   self.class.send(:define_method, "a_#{method}") do |path, *args|
     opts = args.last.is_a?(Hash) ? args.pop : {}
+
     ver = args.first || opts[:api_version] || 1
     fmt = opts[:api_format] || :json
-    a_request(method, Gemfury.endpoint + path).with(
-      :headers => {
-        'Accept' => "application/vnd.fury.v#{ver}+#{fmt}",
-        'X-Gem-Version' => Gemfury::VERSION
-      }
-    )
+    endpoint = opts[:endpoint] || Gemfury.endpoint
+    with_opts = { :headers => { 'Accept' => 'application/vnd.fury.v%s+%s' % [ ver, fmt ],
```

Comment:
Not necessary realignment and whitespace changes for the whole Hash definition here.  Just changing one line from:

    a_request(method, Gemfury.endpoint + path).with(

to

    with_opts  = {

accomplishes the task, and keeps the PR more clear

Reply:
Hi Michael, not sure what you mean your comments above? The suggested code seems to be cut.

Reply:
@markjeee I'm still figuring out the review features - is it possible to highlight a line range for a comment, rather than an individual one?

I was trying to say that the old code was:

```ruby
    a_request(method, Gemfury.endpoint + path).with(
      :headers => {
        'Accept' => "application/vnd.fury.v#{ver}+#{fmt}",
        'X-Gem-Version' => Gemfury::VERSION
      }
    )
```
I would try to keep existing code in place by making it:

```ruby
    with_opts = {
      :headers => {
        'Accept' => "application/vnd.fury.v#{ver}+#{fmt}",
        'X-Gem-Version' => Gemfury::VERSION
      }
    }
```

Reply:
Ahh, got it. Alrighty, updated the commit with the suggestion. :)